Styles.css

```diff
@@ -0,0 +1,199 @@
+* {
+    box-sizing: border-box; 
+    margin: 0;
+    padding: 0;   
+}
+
+html {
+    height: 100%;
+}
+
+
+.navbar {
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    margin: 20px;
+    background-color: antiquewhite; 
+    
+}
+
+
+h1 {
+    color: rgb(5, 147, 147);
+}
+
+h2 {
+    display: flex;
+    color: #00b39b;
+    font: 2.125rem;
+    font-weight: 400;
+    align-content: center;
+    justify-content: center;
+}
+
+.paragraphs{
+    display: flex;
+    align-content: space-between;
+    /* background-color: red; */
+    height: 200px;
+    padding: 20px;
+    font-weight: 100px;
+    align-items: flex-start;
+    font-size: 20px;
+    margin: 20px;
+    justify-content: center;
+    
+}
+
+.p1 {
+    width: 35%;
+    margin-right: 20px;
+    padding: 10px;
+    font-weight: 600;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    
+}
+.p2{
+    width:35%;
+    margin-left: 20px;
+    padding: 10px;
+    font-family: 'Franklin Gothic Medium', 'Arial Narrow', Arial, sans-serif;
+    font-weight: 600;
+}
+.description{
+    display: flex;
+    /* background-color: blueviolet; */
+    justify-content: space-between;
+    margin: 10px 10px 10px 10px;
+    padding: 20px;
+    height: 300px;
+    color:azure;
+    font-size: larger;
+    font-weight: 700;
+}
+
+.card-1{
```

Comment:
Almost all the styles of `card-1`, `card-2`, and `card-3` are the same, this probably indicates that they should have the same class name.